lib/server.js

```diff
@@ -3174,11 +3174,7 @@ class WebServer {
 
         this._listFiles();
 
-        if (this.adapter.config.aliasOnly) {
-            this.adapter.subscribeForeignObjects('alias.0.*');
-        } else {
-            this.adapter.subscribeForeignObjects('*');
-        }
+        this.adapter.subscribeForeignObjects('*');
```

Comment:
I think that this optimisation: 
`if (this.adapter.config.aliasOnly) {
            this.adapter.subscribeForeignObjects('alias.0.*');
        } else {
            this.adapter.subscribeForeignObjects('*');
        }`
could be still helpful. Isn't it?

Reply:
This is actually, what I removed in this change. If only alias.0 is subscribed then onObjectChange is only called for object changes in alias.0 -> if you add / remove / change manual entity settings in an object outside of alias.0, the adapter won't recognize the change and needs to be restarted to reflect the changes. That is why I changed it back to subscribe.

I think it is helpful to restrict automatic device generation to alias.0. But for manual settings it is not a good idea as long as they show up on all objects (and I don't know a way to restrict that). So I wanted to repair manual settings for other objects to work with the old setting. Also it might still be ok to add simple entities like "sensor" or something outside alias.0. 

So now onObjectChange is still called for all object changes but nothing happens if the object is not in alias.0 or is detected to be associated with a manual entity. 
I don't see a way to optimise that further. Or is there another way the adapter is / can be informed about the manual settings on an object? (I don't know much about that part of ioBroker, yet).